builtin/env.c

```diff
@@ -1214,12 +1238,21 @@ void $__init__ () {
         $l$16lambda$methods.__deserialize__ = $l$16lambda$__deserialize__;
         $register(&$l$16lambda$methods);
     }
+    {
+        $WorldAuth$methods.$GCINFO = "$WorldAuth";
+        $WorldAuth$methods.$superclass = ($Super$class)&$value$methods;
+        ;
+        $WorldAuth$methods.__init__ = $WorldAuth$__init__;
+        $WorldAuth$methods.__serialize__ = $WorldAuth$__serialize__;
+        $WorldAuth$methods.__deserialize__ = $WorldAuth$__deserialize__;
+        $register(&$WorldAuth$methods);
+    }
     {
         $Env$methods.$GCINFO = "$Env";
         $Env$methods.$superclass = ($Super$class)&$Actor$methods;
         $Env$methods.__bool__ = ($bool (*) ($Env))$Actor$methods.__bool__;
         $Env$methods.__str__ = ($str (*) ($Env))$Actor$methods.__str__;
-        $Env$methods.__repr__ = ($str (*) ($Env))$Actor$methods.__str__;
+        $Env$methods.__repr__ = ($str (*) ($Env))$Actor$methods.__repr__;
```

Comment:
@sydow you changed the __repr__ method recently, these changes (as generated by actonc) look correct, right? Or should __repr__ on Env, Connection, RFile etc (per below) actualy be __str__?